openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1168,9 +1170,19 @@ A Credential Issuer is identified by a case sensitive URL using the `https` sche
 
 The Credential Issuer's configuration can be retrieved using the Credential Issuer Identifier.
 
-Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`. 
+Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`.
+
+To fetch the Credential Issuer Metadata, a requester MUST send a HTTP GET to the path formed following the steps above utilizing TLS. The Credential Issuer MUST return a JSON document compliant with this specification using the `application/json` media type and a HTTP Status Code 200.
+
+The Wallet is RECOMMENDED to send an Accept-Language Header in the HTTP GET request to indicate the particular languages he is interested in. The Credential Issuer is RECOMMENDED to send a subset of internationalized display data containing the requested languages and using the HTTP Content-Language Header. The Credential Issuer MAY ignore the Accept-Language Header and send all supported languages or a any chosen default subset.
+
+Below is a non-normative example of a Credential Issuer Metadata request:
 
-The path formed following the steps above MUST point to a JSON document compliant with this specification. The document MUST be returned using the `application/json` media type.
+```
+GET /.well-known/openid-credential-issuer HTTP/1.1
+Host: server.example.com
+Accept-Language: fr-CH, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5
```

Comment:
```suggestion
Accept-Language: fr-ch, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5
```

Reply:
according to the codes defined in ISO 639-1

Reply:
The language tags are case insensitive as per https://datatracker.ietf.org/doc/html/rfc3066#section-2.1

Reply:
good catch @jogu, that's why I was asking if we're considering ISO 639-1

It seems to me that in your suggestion you brough RFC3066 as the ref to be included in this PR (I have updated my prev comments accordingly, thx)

> [ISO 3166] recommends that country codes are capitalized (MN Mongolia), while [ISO 639] recommends that language codes are written in lower case (mn Mongolian).

Reply:
We're not considering anything :-) The behaviour of the Accept-language tag is already well defined ( https://datatracker.ietf.org/doc/html/rfc7231#section-5.3.5 ) - we're just using it as it is already defined I believe.

Reply:
@jogu I was asking since I didn't see RFC7231 in the normative or informative specs referenced in this specification

Reply:
I think it is pretty clear that the specification already relies on HTTP 1.1 :-) There is an already a normative reference as a result of the normative reference to OAuth2 (and maybe others) that have normative references to HTTP RFCs. Similarly RFC3066 is a normative reference of HTTP RFC. (I don't know what the rule is on if we need our own normative reference as well.)

Reply:
I learned something, thank you @jogu